docs/upctl_server_create.md

```diff
@@ -6,6 +6,14 @@ Create a server
 upctl server create [flags]
 ```
 
+### Examples
+
+```
+upctl server create --title my_server --zone fi-hel1 --hostname myapp --password-delivery email
```

Comment:
hmm, does the CLI accept e.g. `--title "My Server"`, could hilight that in the example 🤔